samples/v2/add-and-drop-new-database-role.js

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sample-metadata:
+//  title: Add and drop new database role
+//  usage: node add-and-drop-new-database-role.js <INSTANCE_ID> <DATABASE_ID> <PROJECT_ID>
+
+'use strict';
+
+function main(
+  instanceId = 'my-instance',
+  databaseId = 'my-database',
+  projectId = 'my-project-id'
+) {
+  // [START spanner_add_and_drop_database_role]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+  // const projectId = 'my-project-id';
+  // Imports the database admin client
+  const {DatabaseAdminClient} = require('@google-cloud/spanner/build/src/v1');
+
+  // creates an database admin client
+  const databaseAdminClient = new DatabaseAdminClient({
+    projectID: projectId,
+    instanceID: instanceId,
+  });
+
+  async function addAndDropNewDatabaseRole() {
+    // Creates a new user defined role and grant permissions
+    try {
+      const request = [
+        'CREATE ROLE parent',
```

Comment:
Why are you hard-coding the role names? Can't we format it similar to existing samples?

Reply:
Existing samples contains the hard-coded role names, I have followed the same format present in existing samples